### PR TITLE
[EHN] Coordinates 

### PR DESCRIPTION
**Pull request by mts299**
# Scope 

Added two new coordinates AACGM and GEO and changed the name to aid in #151 and #149 

## Approval

**Number of approvals:** 1

## Test

Testing rtp plotting

### CODE REVIEW

**Comment by carleyjmartin** [comment]
I'm getting an error importing Coords
```
File "rtp_test.py", line 2, in <module>
    import pydarn
  File "/Users/carley/Documents/testing_code/venv-coords/lib/python3.7/site-packages/pydarn/__init__.py", line 41, in <module>
    from .utils.radar_pos import radar_fov
  File "/Users/carley/Documents/testing_code/venv-coords/lib/python3.7/site-packages/pydarn/utils/radar_pos.py", line 14, in <module>
    from pydarn import Coords
ImportError: cannot import name 'Coords' from 'pydarn' (/Users/carley/Documents/testing_code/venv-coords/lib/python3.7/site-packages/pydarn/__init__.py)
```

**Comment by mts299** [comment]
@carleyjmartin try now?

File: pydarn/utils/coordinates.py [hunk 1]
```diff
@@ -32,3 +32,5 @@ class Coord(enum.Enum):
 
     RANGE_GATE = enum.auto()
     SLANT_RANGE = enum.auto()
+    GEOGAPHIC = enum.auto()
```
Comment (carleyjmartin) [inline pydarn/utils/coordinates.py:35]:
```suggestion
    GEOGRAPHIC = enum.auto()
```

**Review by carleyjmartin — COMMENTED** [review]
RTP, Summary plots and fov as shown below all working as expected :D

```
aacgm_lats, aacgm_lons=pydarn.radar_fov(96, coords=pydarn.Coords.AACGM, date=dt.datetime(2005,11,26))
```

Just one typo I missed earlier.

**Pull request merged**